### PR TITLE
docs(trace): described usage if interface and href more clearly

### PR DESCRIPTION
**Pull request by markuskeidl**
After some Catena-X internal discussions, we now can describe more clearly how submodel endpoints w.r.t. interface and href should be used. This pull requests documents this.

### CODE REVIEW

File: docs/kits/Traceability Kit/Software Development View/page_software-development-view.md [hunk 1]
```diff
@@ -424,7 +424,7 @@ The actual access information for the EDC is part of the endpoint attribute in t
 {
     "interface": "SUBMODEL-3.0",
     "protocolInformation": {
-        "href": "https://edc.data.plane/{path}",
+        "href": "https://edc.data.plane/{path}/submodel",
         "endpointProtocol": "HTTP",
         "endpointProtocolVersion": ["1.1"],
         "subprotocol": "IDS",
```
Comment (arnoweiss) [inline docs/kits/Traceability Kit/Software Development View/page_software-development-view.md:430]: when referring to CX-0002 R3.2, this should be DSP.

Comment (markuskeidl) [reply docs/kits/Traceability Kit/Software Development View/page_software-development-view.md:430]: Ok. There are several other references to IDS, now DPS. I will update the pull request accordingly.

**Comment by arnoweiss** [comment]
looks good to me.

**Review by mhellmeier — CHANGES_REQUESTED** [review]
@markuskeidl : Can you please update the branch by merging or rebasing the current status of the `docs/trace-release-3.2` into this one?

Further, please update the changelog accordingly.

**Comment by markuskeidl** [comment]
> @markuskeidl : Can you please update the branch by merging or rebasing the current status of the `docs/trace-release-3.2` into this one?
> 
> Further, please update the changelog accordingly.

Done.

**Review by mhellmeier — APPROVED** [review]
LGTM

**Pull request merged**